requirements.txt

```diff
@@ -1,7 +1,7 @@
-numpy
-celery
-PyFlakes
+numpy==1.12.1
+celery==4.1.0
+pyflakes==1.6.0
 taxtastic
```

Comment:
@KyleBehiels Add the version number for taxtastic https://pypi.python.org/pypi/taxtastic